cooltools/lib/checks.py

```diff
@@ -205,15 +212,8 @@ def _is_compatible_cis_expected(
             raise ValueError("expected_df does not look like cis-expected") from e
 
         # Check that view regions are named as in expected table.
-        if not bioframe.is_cataloged(
-            expected_df,
-            verify_view,
-            df_view_col="region1",
-            view_name_col="name",
-        ):
-            raise ValueError(
-                "View regions are not in the expected table. Provide expected table for the same regions"
-            )
+        if verify_view is not None:
+            _is_expected_cataloged(expected_df, verify_view)
 
         # check if the number of diagonals is correct:
         if verify_cooler is not None:
```

Comment:
could you move this inside of the `if verify_view is not None:` branch - as the `verify_cooler` related check relies on `verify_view`

Reply:
hmm-- should we try to create a view from the cooler if `verify_cooler` is passed but `verify_view` is None? or just leave as is?

Reply:
at the very least we'd need to update the docstring that the `verify_cooler` argument also requires a view!

Reply:
> hmm-- should we try to create a view from the cooler if verify_cooler is passed but verify_view is None? or just leave as is?
yeah - probably this is a useful scenario indeed `is_valid_expected(expected_df,..., verify_view=None, verify_cooler=clr)`
I haven't thought about that before ...